src/main/java/com/eternalcode/discordapp/DiscordApp.java

```diff
@@ -1,33 +1,44 @@
 package com.eternalcode.discordapp;
 
-import com.eternalcode.discordapp.command.AvatarCommand;
-import com.eternalcode.discordapp.command.BanCommand;
-import com.eternalcode.discordapp.command.BotInfoCommand;
-import com.eternalcode.discordapp.command.ClearCommand;
-import com.eternalcode.discordapp.command.CooldownCommand;
-import com.eternalcode.discordapp.command.EmbedCommand;
-import com.eternalcode.discordapp.command.KickCommand;
-import com.eternalcode.discordapp.command.PingCommand;
-import com.eternalcode.discordapp.command.ServerCommand;
+import com.eternalcode.discordapp.command.*;
 import com.eternalcode.discordapp.config.DiscordAppConfig;
 import com.eternalcode.discordapp.config.DiscordAppConfigManager;
+import com.eternalcode.discordapp.config.DiscordAppDatabaseConfig;
+import com.eternalcode.discordapp.database.DatabaseManager;
+import com.eternalcode.discordapp.database.repository.RepositoryManager;
 import com.jagrosh.jdautilities.command.CommandClient;
 import com.jagrosh.jdautilities.command.CommandClientBuilder;
 import net.dv8tion.jda.api.JDABuilder;
 import net.dv8tion.jda.api.entities.Activity;
 import net.dv8tion.jda.api.requests.GatewayIntent;
 
 import java.io.File;
+import java.sql.SQLException;
 
 public class DiscordApp {
 
     private static final boolean IS_DEVELOPER_MODE = false;
     private static DiscordAppConfig config;
+    private static DatabaseManager databaseManager;
+    private static DiscordAppDatabaseConfig databaseConfig;
+    private static RepositoryManager repositoryManager;
+
 
     public static void main(String... args) {
         DiscordAppConfigManager configManager = new DiscordAppConfigManager(new File("config"));
         config = new DiscordAppConfig();
+        databaseConfig = new DiscordAppDatabaseConfig();
         configManager.load(config);
+        configManager.load(databaseConfig);
```

Comment:
proponuję zmienić nazwę klasy `DiscordAppDatabaseConfig` na `DatabaseConfig`. Również, można przemyśleć nazwę `DiscordAppConfigManager` na samo `ConfigManager`, w tym przypadku to nie jest tutaj konieczne. A jeśli chodzi o metody load, to mogły by one zwracać config tak samo jak np. w eternalcore, dzięki takiej zmianie będzie dwie linijki mniej. Ale jeśli o to chodzi, to proszę o odpowiedź kogoś bardziej doświadczonego czy jest sens.